support/mock/mockrt.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/filecoin-project/go-address"
 	addr "github.com/filecoin-project/go-address"
 	cid "github.com/ipfs/go-cid"
-	mh "github.com/multiformats/go-multihash"
+	"github.com/multiformats/go-multihash"
```

Comment:
Restore the alias, or alias as `multihash` explicitly since it doesn't match path.

Reply:
Done. (although dropping the `go-` prefix is pretty common, IMO)